editor/header/mode-switcher/style.scss

```diff
@@ -1,64 +1,36 @@
 .editor-mode-switcher {
 	position: relative;
 	margin-right: $item-spacing;
-}
-
-.editor-mode-switcher__toggle {
+	padding-right: $item-spacing;
 	color: $dark-gray-500;
-	padding: $item-spacing;
 	align-items: center;
 	cursor: pointer;
-	border: none;
-	background: none;
 	border-right: 1px solid $light-gray-500;
 	outline: none;
 	display: flex;
 	align-items: center;
-}
 
-.editor-mode-switcher__content {
-	position: absolute;
-	top: 40px;
-	border: 1px solid $light-gray-500;
-	color: $dark-gray-500;
-	min-width: 100px;
-
-	button {
-		padding: 8px;
-		display: block;
-		border: none;
-		background: white;
+	select {
+		background: transparent;
+	    line-height: 1;
```

Comment:
No linting to capture it, but we ought to be cautious of mixed spacing in SCSS files.